dashboard/app/models/script.rb

```diff
@@ -333,7 +333,7 @@ def beta?
   end
 
   def self.beta?(name)
-    name == 'edit-code' || name == 'gradeKinder' || name == 'grade1' || name == 'grade2' || name == 'grade3' || name == 'grade4' || name == 'grade5'
+    name == 'edit-code' || name == 'gradeKinder' || name == 'grade1' || name == 'grade2' || name == 'grade3' || name == 'grade4' || name == 'grade5' || name.start_with?('csd')
```

Comment:
I don't see a problem with something like csd_test being marked as beta, is there anything that being beta does other than putting Beta in the header?